.gitignore

```diff
@@ -1,3 +1,16 @@
 .env
 node_modules/
 dist/
+
+## used for some private servers, dist/**.js concated into build
+# make shure requrie.js is not concated and main.js is concated at last
+build/
+
+## favorite idea
+.idea
```

Comment:
I think this should be in your global .gitignore file. I usually follow the rule:
Is it related to my environment: global .gitignore
Is it related to this specific project: local .gitignore